doc/nrf/getting_started/installing.rst

```diff
@@ -343,6 +342,14 @@ This is a simplified structure preview.
 There are additional folders, and the structure might change over time.
 The full set of repositories and folders is defined in the manifest file.
 
+In order to change the state of development you want to use to a different revision, enter the following commands from the :file:`nrf` directory you created earlier::
+
+  git checkout *next_revision*
+  west update
+
+Note that in this case, *next_revision* can be a SHA (for example, ``224bee9055d986fe2677149b8cbda0ff10650a6e``) in addition to a branch or tag name.
+You can use any revision of the sdk-nrf repository that is available on your local file system.
```

Comment:
Some of this information is already mentioned in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/getting_started/updating.html#examples-of-commands 

As the new addition is not part of the procedure, I would suggest adding it to the updating page, not here.

Reply:
Updated both